### DISCUSSION TITLE
Two Folders: same content.

### DISCUSSION THREAD

**Discussion by ghost** [abraunegg/onedrive#2781] [Q&A] [answered]
Hi! I'm using the client. At the present time I have in my home directory two folders related to OneDrive: /home/OneDrive and /home/OneDriveusername. Checking the GUI program, I can see that the one that is being synchronized is /home/OneDrive, so I do not know what to do with the second one. Is there a method or something I can do to verify the /home/OneDriveusername directory is now just a folder with the content downloaded but not in sync so that I could delete the whole folder without risking to lose data? Thanks a lot!

**Comment by abraunegg** [comment, marked as answer]
The question would be when was `/home/OneDriveusername` created? What is its time stamp and when was content in it last modified.

This client uses the 'sync_dir' configuration option to determine what path is used to sync the data to.

All configurations are stored in `~/.config/` .. usually under the 'onedrive' folder - but if you have multiple configurations, you will have multiple configuration folders.

If you *only* have one configuration folder, then I would validate that the application, when using `onedrive --display-config` matches your current environment.

You can also use `rsync` with the `--dry-run` command to to simulate syncing the contents of one folder to another. This will show you the differences between the folders without actually copying or deleting any files.
```
rsync -av --dry-run --delete /home/OneDrive/ /home/OneDriveusername
```

This command will list the files that would be deleted or copied to make `/home/OneDriveusername` identical to `/home/OneDrive/`. If rsync shows that all the files in `/home/OneDriveusername` are present in `/home/OneDrive/` and are newer, then `/home/OneDriveusername` likely contains only older content.


Between all of these checks you can determine if `/home/OneDriveusername` is safe to remove.

**Reply by ghost** [reply]
Okey, solved it! Thank you so much!
